src/componentSelectorRule.ts

```diff
@@ -46,7 +47,7 @@ export class Rule extends SelectorRule {
     },
     optionsDescription: Utils.dedent`
       Options accept three obligatory items as an array:
-      1. \`${OPTION_ELEMENT}\` or \`${OPTION_ATTRIBUTE}\` forces components either to be elements or attributes.
+      1. \`${OPTION_ELEMENT}\` or \`${OPTION_ATTRIBUTE}\` forces components either to be elements or attributes. It can be both (wrapped in an array, see example #4)
```

Comment:
Would you change this to: `forces components to be used as either elements, attributes, or both (not recommend)`.